plugins/chain_plugin/include/eos/chain_plugin/chain_plugin.hpp

```diff
@@ -160,7 +160,7 @@ class read_only {
       name        scope;
       name        code;
       name        table;
-//      string      table_type;
+
```

Comment:
`table_type` is used to specify the whether the table is `Dual Index Table` or `Single Index Table`

Reply:
Accept